llvm/test/CodeGen/AArch64/hwasan-check-memaccess-fixedshadow.ll

```diff
@@ -0,0 +1,144 @@
+; RUN: llc --hwasan-mapping-offset=4398046511104 < %s | FileCheck %s
+
+target triple = "aarch64--linux-android"
+
+define ptr @f1(ptr %x0, ptr %x1) {
+  ; CHECK: f1:
+  ; CHECK: str x30, [sp, #-16]!
+  ; CHECK-NEXT: .cfi_def_cfa_offset 16
+  ; CHECK-NEXT: .cfi_offset w30, -16
+  ; CHECK-NEXT: bl __hwasan_check_x1_1
+  ; CHECK-NEXT: mov x0, x1
+  ; CHECK-NEXT: ldr x30, [sp], #16
+  ; CHECK-NEXT: ret
+  call void @llvm.hwasan.check.memaccess.fixedshadow(ptr %x1, i32 1)
```

Comment:
Because you have no these intrinsics, you can't precommit as-is
so try precommit with lvm.hwasan.check.memaccess() version and the same fixed shadow address

Reply:
Added https://github.com/llvm/llvm-project/pull/89328